test/wrappers/ortools_test.rb

```diff
@@ -5368,4 +5368,28 @@ def test_relations_sent_to_ortools_when_different_lapses
       end
     }
   end
+
+  def test_always_one_time_window_provided_for_rests
+    problem = VRP.basic
+    problem[:rests] = [{ id: 'rest', duration: 10 }]
+    problem[:vehicles][0][:rest_ids] = ['rest']
```

Comment:
This instance doesn't lead to a segfault (even without the fix introduced in the PR) because the pause is eliminated by the simplification. 

Adding a TW to one services leads to the segfault (the rest of the test would need a correction accordingly): 
```suggestion
    problem = VRP.basic
    problem[:rests] = [{ id: 'rest', duration: 10 }]
    problem[:vehicles][0][:rest_ids] = ['rest']
    problem[:services][0][:activity][:timewindows] = [{ start: 0, end: 20 }] # so that rest is not simplified
```